CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ The changelog for `MessageKit`. Also see the [releases](https://github.com/Messa
 
 ## 3.0.0
 
+## Upcoming Release
+
+### Added
+
+- Add support for styling NSLinkAttribute with existing urlAttributes in MessageLabel. [#1091](https://github.com/MessageKit/MessageKit/pull/1091) by [@marcetcheverry](https://github.com/marcetcheverry)
+
+## Upcoming Release
```

Comment:
Duplicate header here

Reply:
Yeah sorry about that, it is hard to keep track of all the CHANGELOGs with multiple branches and the one master integration branch I keep. It would be easier if the CHANGELOGs were only changed by maintainers on merge.